python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -111,9 +108,44 @@ def __init__(
         # Task for self.read_loop
         self.read_loop_task = salobj.make_done_future()
 
-        # Most recent Rotator Application telemetry sample,
-        # or None if camera cable wrap not tracking.
-        self.prev_rot_application = None
+        # Most recent Rotator Application telemetry sample information,
+        # or None if no information from the rotator was ever received.
+        self.rotator_time = None
+        self.rotator_position = None
+        self.rotator_velocity = None
+        self.rotator_demand = None
```

Comment:
You use this same pattern in 2 places and I very strongly feel it is much better to store *one* item that has all the data because that avoids any risk that the separate fields will be out of synch with each other, or that some code change will fail to set one of them.

(I am fine with your use of `self.ccw_demand_position` and `self.ccw_demand_velocity` because it's just two related values, it's not being copied from a struct that has all the info, and they are set in somewhat separate places. That said, even in that case a single object would be slightly safer.)

For `self.rotator_x` you not need store anything. For this field you can call `self.rotator.tel_Application.get(flush=False)`. This will return `None` before telemetry data has been seen and will have the most recently seen value after that. There is slight overhead because `get` now checks that the Remote has started. But I suggest living with it to avoid needlessly duplicating data.

In the case of the camera cable wrap data (the `ccw_x` attributes) I see no reason to have a callback function at all. Just call `self.mtmount.tel_Camera_Cable_Wrap.get(flush=False)` whenever you want the current value. That will save a lot of CPU cycles since you avoid the overhead of the callback function.